README.md

```diff
@@ -75,6 +77,43 @@ generic with minimal need for project-specific behavior.
 [Docker]: https://docs.docker.com/install/overview/
 
 
+### 2.1.1 git-secrets
+
+[git-secrets] helps prevent secrets (passwords, credentials, etc.) from being
+submitted into a Git repository. See the *Installing git-secrets* section of the
+project's README for instructions how to install [git-secrets] on your OS.
+
+Once installed, git-secrets will need to be configured for your local repository
+clones. To configure git-secrets in a single clone, run:
+
+    cd /path/to/my/clone
+    git secrets --install
+    git secrets --register-aws
+    
+To automatically configure git-secrets in all future clones, run:
+
+    git secrets --install ~/.git-templates/git-secrets
+    git config --global init.templateDir ~/.git-templates/git-secrets
+    git secrets --register-aws --global
+
+Finally, once configured you should verify the functionality of git-secrets.
+With a clean working copy, run these commands in each of your clones to verify a
+successful setup:
+
+    cd /path/to/my/clone
+    echo -e 'AWS_ACCOUNT_ID=0000-0000-000\x30' > git_secrets_test_file.txt
+    git add git_secrets_test_file.txt
+    .git/hooks/pre-commit
```

Comment:
You must not have seen https://github.com/DataBiosphere/azul/pull/4889#discussion_r1084754201

Reply:
`git hook` is only available in newer versions of `git`.

```
(.venv) daniel@Crispin ~/repo/azul2 $ git hook
git: 'hook' is not a git command. See 'git --help'.
(.venv) daniel@Crispin ~/repo/azul2 $
(.venv) daniel@Crispin ~/repo/azul2 $ git --version
git version 2.30.1 (Apple Git-130)
```

From: https://stackoverflow.com/questions/55959023/call-githooks-as-standalone-scripts/71101716#71101716
> This will be officially supported with Git 2.36 (Q2 2022)

Should we add a minimum version of `git` to the `Development Prerequisites` section of the README?